internal/test/e2e/e2e_test.go

```diff
@@ -547,31 +528,17 @@ func webhookTests(t *testing.T, kubectl *kubectlContext) {
 }
 
 func sampleClusterTests(t *testing.T, kubectl *kubectlContext, sampleClusterPath string) {
-	err := kubectl.Apply(
-		"--filename", filepath.Join(*fRepoRoot, "internal", "test", "e2e", "fixtures", "cluster-client-service.yaml"),
```

Comment:
This service manifest allowed external access to the etcd cluster. It's no longer needed and I've deleted the file.